### PR TITLE
Create LIT7070Qene.xml

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I have recorded another Qǝne from BNFabb145 (f. 55vb). 

As for the translation of the title, I am not sure that I have interpreted ba-mǝsleka correctly. Furthermore, I am uncomfortable with ignoring the accusative in ḥaqla, which may have happened because of the rhyme.

### CODE REVIEW

File: new/LIT7070Qene.xml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7070Qene" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">">ተሠይጠ፡ ዮሴፍ፡ በምሳሌከ፡ አባግዓ፡ ያዕቆብ፡ ይርአይ፡ መንገለ፡ ወፈረ፡ ሐቅለ።</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Taśayṭa Yosef ba-mǝssāleka ʾabāgʿā Yāʿqob yǝrʾay mangala wafara ḥaqla (Qǝne of the kʷǝllǝkomu-type)</title>
+                <title xml:lang="en" corresp="#t1">Joseph was sold in your likeness, while tending Jacob's sheep in the field …</title>
```
[the remark this answers is not in the document]
Comment (karljonaskarlsson) [reply new/LIT7070Qene.xml:10]: (I wonder if the -ka in በምሳሌከ might refer to Christ. In the Dǝggʷā, I seem to remember cases where unspecified pronouns refer to God.)

**Review by karljonaskarlsson — APPROVED** [review]

File: new/LIT7070Qene.xml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Qǝne of the kʷǝllǝkomu-type. Type and author are not indicated in the manuscript.</p>
```
[the remark this answers is not in the document]
Comment (karljonaskarlsson) [reply new/LIT7070Qene.xml:37]: (kʷǝllǝkǝmu)

[the remark this answers is not in the document]
Comment (CarstenHoffmannMarburg) [reply new/LIT7070Qene.xml:37]: I wonder, why the keyword is Kwellekomu then?

File: new/LIT7070Qene.xml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-09-12">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="BNFabb145"/> on f. 55vb. Division into verses is indicated with
+                    <foreign xml:lang="gez">።</foreign>. The end of the poem is indicated with <foreign xml:lang="gez">። ።</foreign>.</note>
```
[the remark this answers is not in the document]
Comment (CarstenHoffmannMarburg) [reply new/LIT7070Qene.xml]: I have not seen any keyword "punctuation" to be included in `<term key="....">።</term>`.

**Review by karljonaskarlsson — APPROVED** [review]

**Review by eu-genia — APPROVED** [review]

**Pull request merged**